torch_xla/csrc/runtime/env_hash.h

```diff
@@ -0,0 +1,15 @@
+#include <torch/csrc/lazy/core/hash.h>
+
+namespace torch_xla {
+namespace runtime {
+namespace hash {
+
+// Take a hash of XLA flags which impact the compilation result.
+// TODO(jonbolin): We should move away from manually hashing the env vars and
+// instead hash the compilation environment directly when the functionality is
+// supported in the runtime.
+torch::lazy::hash_t HashXlaEnvVars(bool force_rehash = false);
```

Comment:
What's `force_reshash`?

Reply:
This is for testing - the hash is statically stored, but in the tests I need to be able to change the environment and rehash.

I suppose I'm actually storing the result of the environment hash within the computation client, so this is still only called once. I'll make the hash results here non-static.